### PR TITLE
MEN-3961 Additional fixes of text around the links to the docs

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,7 +101,8 @@ sudo make install
 ### Installation notes
 
 Installing this way does not offer a complete system updater. For this you need
-[additional integration steps](https://docs.mender.io/devices). However, it is
+additional integration steps: [in Yocto case](https://docs.mender.io/system-updates-yocto-project)
+or [for Debian family](https://docs.mender.io/system-updates-debian-family). However, it is
```
Comment (mirzak) [inline README.md:105]: Suggestion:

```
For this you need additional integrations steps, depending in which OS you are using consult one of the following:

- [System updates: Debian family](https://docs.mender.io/system-updates-debian-family)
- [System updates: Yocto Project](https://docs.mender.io/system-updates-yocto-project)
```

File: cli/snapshot.go [hunk 2]
```diff
@@ -35,8 +35,11 @@ const (
 	errMsgDataPartFmt = "Device-local data is stored on the partition " +
 		"being snapshotted: %s. The recommended approach is to have " +
 		"a separate data-partition mounted on \"/data\" and add a " +
-		"symbolic link (%s -> /data). https://docs.mender.io/devices/" +
-		"general-system-requirements#partition-layout"
+		"symbolic link (%s -> /data). Consult either " +
+		"https://docs.mender.io/system-updates-debian-family/overview#partition-layout" +
+		"for Debian family or " +
+		"https://docs.mender.io/system-updates-yocto-project/overview#partition-layout" +
+		"inca se you use Yocto"
```
Comment (mirzak) [inline cli/snapshot.go:42]: `"inca se you use Yocto"`  -> `"in case you use the Yocto Project`

**Review by mirzak — CHANGES_REQUESTED** [review]

File: cli/snapshot.go (hunk 2 above)
Comment (merlin-northern) [reply cli/snapshot.go]: (I really have a malfunction with the space key, sorry about that)

File: README.md (hunk 1 above)
Comment (merlin-northern) [reply README.md]: sure!

**Review by mirzak — APPROVED** [review]

**Comment by merlin-northern** [comment]
stopped the pipelines, they passed previously and the change after review was cosmetic.

**Pull request merged**